### PR TITLE
Copyright message

### PR DESCRIPTION
**Pull request by OHaiDanny**
Issue #108. Let me know if this is the right spot for the params. I'm wondering if this is really needed to be in the `balance-static` repo. Will it read the params once everything is merged?

### CODE REVIEW

File: layouts/partials/footer.html [hunk 1]
```diff
@@ -33,7 +33,7 @@ <h3 class="footer-header pb-1">
       {{ end }}
     </div>
     <div class="py-5 footer-copyright">
```
Comment (nonowazu) [inline layouts/partials/footer.html:35]: Maybe put a

```html
{{ with .Site.Params.copyrightMessage }}
<div class="py-5 footer-copyright">
<!-- the rest -->
{{ end }}
```

**Review by nonowazu — CHANGES_REQUESTED** [review]
Just oooone small change

File: layouts/partials/footer.html (hunk 1 above)
Comment (OHaiDanny) [reply layouts/partials/footer.html:35]: will do!

Comment (OHaiDanny) [reply layouts/partials/footer.html]: with broke the site. but If statement works :)

**Comment by OHaiDanny** [comment]
I also fixed some of the footer margin spacing. just some small CSS changes

File: layouts/partials/footer.html (hunk 1 above)
Comment (nonowazu) [reply layouts/partials/footer.html]: Interesting, but quick fix if we get it to work with `with` later.

**Pull request merged**